src/Todo/Model.hs

```diff
@@ -31,4 +49,32 @@ data Todo = Todo
     , todoSubtasks    :: Todos
     , todoDeadline    :: Maybe Deadline
     , todoDescription :: Maybe String
-    } deriving (Eq, Show)
+    } deriving (Eq, Show, Generic)
+
+instance Hashable Todo 
+
+defTodo :: Title -> Priority -> Tags -> Todo
+defTodo todoTitle todoPriority todoTags = Todo { todoState = TODO
+                                               , todoSubtasks = HSet.empty
```

Comment:
`HashSet` is an instance of `Monoid` so you can just write `mempty` instead of `HSet.empty`. But if you prefer monomorphic versions it's okay to leave it as it is.

Reply:
It is just a bit more readable to me. If I saw `mempty` in someone else's code I would have to go and check `Monoid` instance for this type (who knows how is it implemented).